packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -596,13 +646,19 @@ abstract class DiagnosticsNode {
   ///
   /// The [style] argument must not be null.
   ///
+  /// At most one of `hidden` and [level] should be specified. Setting `hidden`
+  /// to `true` is equivalent to setting [level] to [DiagnosticLevel.hidden].
+  /// If neither `hidden` or [level] is set, [level] defaults to
+  /// [DiagnosticsLevel.info].
```

Comment:
am i missing something? I don't see a `hidden` on this constructor.

Reply:
Removed all references to `hidden`